package.json

```diff
@@ -30,6 +30,6 @@
     "lodash-es": "4.17.15"
   },
   "devDependencies": {
-    "vue-template-compiler": "2.6.12"
+    "vue-template-compiler": "^2.6.12"
```

Comment:
Why are you updating this?

Reply:
not sure how this changed, I'll bring it back.

Reply:
Updated and fixed conflicts